app/assets/javascripts/application.js

```diff
@@ -1,6 +1,12 @@
 //= require require_config
 
-require(['componentLoader'], function(componentLoader) {
+require(['componentLoader', 'jquery'], function(componentLoader) {
```

Comment:
Should add the $ here so you know you're using the jquery you just pulled in. So:

```js
require(['componentLoader', 'jquery'], function(componentLoader, $) {
```